### ISSUE TITLE
[FEATURE] Agent output as JSON format (Sequential Agents)

### ISSUE THREAD

**Issue by rb58853** [FlowiseAI/Flowise#3114]
**I want to use the output of a Agent Node as JSON format**
I am using `Sequential Agents` , I have an `Agent Node` with otput, for example, `{'var1': 'value1', 'var2':'value2'}`. I want to use this output value (node output) as JSON format for using this in input params of a `Custom JS function`!

I need something like `Structured Output Perser` that i can use in an `Agent Node`.

**Comment by HenryHengZJ** [comment]
2 ways you can solve this:

1.) Specify in system message to ask LLM returns a JSON. and give few examples.
2.) Use LLM with JSON structured output:
![image](https://github.com/user-attachments/assets/8761aa42-767f-4290-a066-fe300acbecb0)

**Comment by rb58853** [comment]
I have tried both alternatives, but neither recognizes the output as JSON. The output is a JSON, but it cannot be passed as input to a Custom JS Function because it is not recognized as JSON, but rather as LLM output or Agent output.

**Comment by HenryHengZJ** [comment]
yeah currently custom JS function is not possible to be connected to agentflows, will add that support

**Comment by marfal** [comment]
+1 
Or please let us manually enter JSON into the JSON Structured Output through a simple text box like in Advanced Structured Output Parser node that maybe could be parsed (no pun intended) by Flowise into the table-display structure that's currently in place. Adding key-value pairs using the "add item" is very cumbersome when dealing with more then just few elements. Thank! :)
![image](https://github.com/user-attachments/assets/f6ce4ce3-df79-4326-a299-991bf7cf5055)

**Comment by dentro-innovation** [comment]
I'd also like to have the tool agents and conversational chains being able to use the json output parser!

**Comment by pietrobonomo** [comment]
I would also like the tool agents and conversational chains to be compatible with the JSON output parser—it would be a game changer for me!

**Comment by thiagolealassis** [comment]
I’m thrilled about the potential of implementing JSON-formatted outputs for agent responses in Flowise! This enhancement is a game-changer, enabling seamless integration with custom JavaScript functions and empowering developers to craft more dynamic and efficient conversational agents.

By adopting this feature, Flowise will not only streamline workflows but also position itself at the forefront of AI development tools. Let’s make this happen and elevate Flowise to new heights! 🚀

**Comment by dasheck0** [comment]
Also interested in this feature. When it is easy to configure agents to output JSON it opens up a whole new world where this can be included in automation pipelines, used as API and so on.

**Comment by CuriousGu** [comment]
I'd also like to have the tool agents being able to use the json output parser!
